Rubberduck.Core/UI/Settings/SettingsForm.cs

```diff
@@ -12,6 +12,7 @@ namespace Rubberduck.UI.Settings
     public interface ISettingsFormFactory
     {
         SettingsForm Create();
+        SettingsForm Create(SettingsViews activeView);
```

Comment:
I'm probably being oblivious -- where's the implementation?

Reply:
I thought (maybe I`m wrong, but I tested it and it works) that the IoC is passing the `activeView ` into the `SettingsForm` constructor where it`s passed to the `SettingsControlViewModel` constructor where it is set here `SelectedSettingsView = SettingsViews.First(v => v.View == activeView);
`